package.json

```
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "@exodus/bitcoinerlab-secp256k1": "^1.0.5-exodus.1",
+    "@noble/secp256k1": "1.7.1",
```

This should probably match the semver specifier used by `@bitcoinerlab/secp256k1`, to avoid duplication of the dependency.

```suggestion
    "@noble/secp256k1": "^1.7.1",
```